nebula/src/nebula/gateway/grpc_server.py

```
@@ -0,0 +1,23 @@
+from nebula.gateway.faq_handler import FAQServiceHandler
+
+
+logger = logging.getLogger("nebula.gateway")
+logging.basicConfig(level=logging.INFO)
+
+def serve():
+    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+
+    # FAQ-Handler registrieren
+    faq_pb2_grpc.add_FAQServiceServicer_to_server(FAQServiceHandler(), server)
+    #(später) Weitere Handler registrieren
```

Flake8 (7.2.0)

[error] block comment should start with '# '

(E265)